ansible/files/bootstrap/telemetry.sh

```diff
@@ -209,6 +221,35 @@ rm RRO-3.2.1-el6.x86_64.tar.gz
 cd RRO-3.2.1; sudo ./install.sh; cd $HOME
 $ANACONDA_PATH/bin/pip install rpy2
 
+# Setup metrics reporting
+if [ "$METRICS_PROVIDER" = "datadog" ] ; then
+    # Pull down and decrypt Datadog API key; see https://github.com/mozilla/emr-bootstrap-spark/pull/485
+    DD_DIR=$(mktemp -d)
+    aws s3 cp $TELEMETRY_CONF_BUCKET/packages/sops-3.0.4-1.x86_64.rpm $DD_DIR/
+    aws s3 cp $TELEMETRY_CONF_BUCKET/credentials/emr.yaml $DD_DIR/
+    sudo yum -y localinstall $DD_DIR/sops-3.0.4-1.x86_64.rpm
+    export DD_API_KEY=$(/usr/local/bin/sops --decrypt --extract '["datadog_agent::api_key"]' $DD_DIR/emr.yaml)
+    # Install datadog-agent; see https://app.datadoghq.com/account/settings#agent/aws
+    bash -c "$(curl -sLS https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
```

Comment:
I prefer using the vendor's install because they can update it without us having to make changes. It would be a good idea to add some monitoring to ensure that the agent is actually installed/running afterwards.

https://docs.datadoghq.com/agent/basic_agent_usage/ubuntu/?tab=agentv6
Do we want to set DD_UPGRADE=true to install v6?